### PR TITLE
Only specify GL profile on macos

### PR DESCRIPTION
**Pull request by torkleyy**
cc @msiglreith That should improve compatibility if I understood you correctly?

## Compatibility checked

(Please use just some example that opens a window and draws something)

* [x] Linux, Mesa
* [x] Linux, Nvidia prop
* [x] OS X
* [x] Windows

### CODE REVIEW

**Comment by jojolepro** [comment]
Working on linux nvidia x11

**Comment by Aceeri** [comment]
Works fine on OS X.

File: amethyst_renderer/src/renderer.rs [hunk 1]
```diff
@@ -294,13 +294,15 @@ fn init_backend(wb: WindowBuilder, el: &EventsLoop, config: &DisplayConfig) -> R
...
+    #[cfg(target_os = "macos")]
+    use glutin::{GlProfile, GlRequest};
 
     let ctx = glutin::ContextBuilder::new()
         .with_multisampling(config.multisampling)
-        .with_vsync(config.vsync)
-        .with_gl_profile(GlProfile::Core)
-        .with_gl(GlRequest::Latest);
+        .with_vsync(config.vsync);
+        #[cfg(target_os = "macos")]
```
Comment (Rhuagh) [inline amethyst_renderer/src/renderer.rs]: tabbed.

Comment (torkleyy) [reply amethyst_renderer/src/renderer.rs]: tabbed?

Comment (Rhuagh) [reply amethyst_renderer/src/renderer.rs]: Indentation is off

Comment (torkleyy) [reply amethyst_renderer/src/renderer.rs]: Oh wow how did I not see this..

**Pull request merged**